src/encoded/static/components/typeutils.js

```diff
@@ -295,44 +295,39 @@ export function fileStatusList(session, sessionProperties) {
 }
 
 
-// Display supersedes/superseded_by lists.
-export const Supersede = ({ context }) => {
+/**
+ *  Display supersedes/superseded_by/alternate_accessions lists.
+ */
+export const ReplacementAccessions = ({ context }) => {
```

Comment:
Not a big deal
Maybe it is just me, but I see nothing really being replaced.

Reply:
Accessions are being replaced by other accessions — alternates, supersedes.